src/PeriodActions.php

```diff
@@ -8,51 +8,50 @@
 
 trait PeriodActions
 {
-    public function forLastMonth(): static
+    private const UNIT_HOURS = 'hours';
```

Comment:
Please run `composre csf` to adjust your changes to our company's codestyle ruleset.